### ISSUE TITLE
Toggle button icon

### ISSUE THREAD

**Issue by zacekjakub** [StreamController/DeckPlugin#5]
Hello, I would like to change a button icon either with API or at least with toggle action after it calls some command, is that possible or a planned feature?

Thank you!

**Comment by Core447** [comment]
Unfortunately this is currently not possible. The next release will include states allowing you to have different states per key that you can change after the command. I might also add a way to change the state of buttons from external scripts through a command like `flatpak run com.core447.StreamController --change-state DECK_SERIAL 0x0 2`

**Comment by Jost-Philip** [comment]
@Core447 : Is there any new development on the "--change-state DECK_SERIAL 0x0 2" feature mentioned above?
For me, this would be the last piece of the puzzle missing to make the Stream Deck universally usable in Linux in my setup!
It would mean that you can easily and quickly implement any missing logic externally, what ever it might be, and synchronize the Stream Deck with external states.

If "the next app version" is already out there: where can I get it?
If now: what is missing, and how can I help?

Thanks!
Jost

**Comment by zacekjakub** [comment]
The project seems to be discontinued.

**Comment by Jost-Philip** [comment]
Yeah... the plugins store seems to be gone, too.
I'm digging through the sources as we speak, but my Python is a little rusty, and Core447 doesn't seem to be a big fan of code comments, so it's going to take me a while to reverse-engineer the logic and find the right places to add things...

**Comment by zacekjakub** [comment]
Seems like a project: https://github.com/StreamController/StreamController

has brighter future. :)

**Comment by Jost-Philip** [comment]
yes, that's  where the logic is actually implemented. The DeckPlugin just seems to provide configuration access to the triggers, so one can control deck functions with the deck buttons etc.
I'll open a new issue there as soon as I find the time.

**Comment by Jost-Philip** [comment]
see https://github.com/StreamController/StreamController/issues/381
